sdk/storage/test-resources.json

```diff
@@ -407,6 +446,14 @@
       "type": "string",
       "value": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('dataLakeAccountName')), variables('mgmtApiVersion')).keys[0].value]"
     },
+    "STORAGE_ENDPOINT_SUFFIX": {
+        "type": "string",
+        "value": "[parameters('storageEndpointSuffix')]"
+    },
+    "AZURE_AUTHORITY_HOST": {
```

Comment:
I don't think this is required. Net-TestResources already has these variables as the output.